source/leuven/genius_2_rocky.rst

```diff
@@ -0,0 +1,116 @@
+If you only have a Conda environment working on Genius, it's best to create a new Conda installation. In this case, it is recommended to recreate your environment for full compatibility with the new OS. Best practice is to choose a new installation folder with explicit mention of the new OS, e.g. ::
+
+   ${VSC_DATA}/miniconda3-rocky
+  
+You can then use this Conda environment after the migration. You can prepare this on the test nodes.
+
+
+.. _impact_on_starting_jobs:
+
+Impact on starting jobs on Genius and wICE
+------------------------------------------
+In order to minimize the changes you need to make to your jobscripts, an appropriate module path (``$MODULEPATH``) will be set by default at the start of your job of the the migration of Genius to Rocky 8 OS. This new module path will now contain all toolchain versions, starting from 2018a on Genius and starting from 2021a on wICE. This is an important change! Previously the modulepath was set to a single toolchain, that would not change over time. You might have set a module path in your jobscripts to refer to newer toolchains.
+
+.. note::
+
+   If you have set a module path explicitly in your jobscript, you can remove it from your jobscript or change it to the module path for Rocky 8.
+
+
+
+Using cluster modules
+~~~~~~~~~~~~~~~~~~~~~
+
+Instead of working with a single default, unchanging, modulepath, refering to a single default toolchain, a new approach is taken. Each cluster partition will have a so called cluster module that sets the ``$MODULEPATH`` that is valid for the specific nodes in the cluster partition. The cluster module will detect the underlying CPU architecture and uses this for setting the path correctly.
+
+
+.. _check_available_software:
+
+Check available software
+~~~~~~~~~~~~~~~~~~~~~~~~
+On the login node you will be able to load the different available cluster modules (only for experimentation, not for actual computing). In order to do this you can query the available cluster moduels with::
+
+   $ module avail
+
+this wil show the available cluster modules::
+
+   --------------------------- /apps/leuven/etc/modules ---------------------------
+   cluster/default
+   cluster/genius/amd_long               (S)
+   cluster/genius/amd                    (S)
+   cluster/genius/batch_debug            (S)
+   cluster/genius/batch_long             (S)
+   cluster/genius/batch                  (S)
+   ...
+   cluster/wice/batch
+   ...
+
+Loading any of this modules on the login node::
+
+   $ module load cluster/genius/batch
+
+will set the module path for the modules that are applicable for the Genius ``batch`` partition::
+
+   $ module load cluster/wice/batch
+
+will set the the module path of the wICE ``batch`` partition. When you do this on the login node you can examine wich modules are available with the regular commands, e.g.::
+
+   $ module avail
+   $ module spider cp2k
+
```

Comment:
It does not mean that these modules are available and could be used on login nodes.

Reply:
I hope that was clear from the "(only for experimentation, not for actual computing)". Rest of the coments I've added.